whmcs-ruby.gemspec

```diff
@@ -16,7 +16,8 @@ Gem::Specification.new do |s|
   s.files           += Dir['lib/**/*']
   s.files           += Dir['test/**/*']
 
-  s.add_dependency('crack', '= 0.1.8')
+  s.add_dependency('crack', '= 0.4.3')
+  s.add_dependency('htmlentities', '= 4.3.4')
```

Comment:
It'd be nicer not to bind to a particular version as it means that the gem can't be used in a project which uses any other version of these gems. Can we be a bit more permissive on this dependency?